projects/igniteui-angular/src/lib/snackbar/snackbar.component.html

```diff
@@ -1,6 +1,7 @@
 <div class="igx-snackbar" *ngIf="isVisible" (@slideInOut.start)="snackbarAnimationStarted($event)" (@slideInOut.done)="snackbarAnimationDone($event)"
     [@slideInOut]="isVisible">
     <span class="igx-snackbar__message" [@fadeInOut]="isVisible">{{ message }}</span>
+    <ng-content *ngIf="!message"></ng-content>
```

Comment:
I am not sure if we should refrain from showing the content when a message is specified. Although, they can easily add some text content along with the other content they need in. @kdinev , any thought?
I believe, however, we should ngIf the message span, especially now that we can have content without a message!

Reply:
I made the changes suggested by @gedinakova.